src/components/App.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './App.scss';
+import Routes from '../routes';
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      appName: '',
+      teamName: ''
+    };
+  }
+
+  componentDidMount() {
```

Comment:
@NkFab Why do you need componentDidMount? Can't you just provide the default value when initializing the state?